core-bundle/src/EventListener/CsrfTokenCookieSubscriber.php

```diff
@@ -145,6 +145,7 @@ private function replaceTokenOccurrences(Response $response): void
         }
 
         $response->setContent($content);
+        $response->headers->remove('Content-Length');
```

Comment:
Might want to add a comment explaining why this is done?

Reply:
Added in 43318278741447f580003c58e803aebab9d80587.